features/FxA-107-security-event-history/README.md

```diff
@@ -0,0 +1,74 @@
+# Security Events History
+
+## Problem Summary
+
+As a Firefox Accounts user, I want to reduce the amount of sign-in
+confirmations I must make, especially when I am logging in from the same
+locations each time (my office and/or home).
+
+--------
+
+## Outcomes
+
+To start, report whenever a sign in attempt that would trigger signin
+confirmation could have been elided, based on correlating the current
+signin request with the user's history.
+
+## Metrics
+
+We can log an info message each time this would occur, tagged as
+`Account.history.seen`. A graph can be created, comparing:
+
+- Number of signin requests
+- Number of `Account.history.seen`
```

Comment:
Even at this early stage, it would be useful to see if we can detect potential false-positives or false-negatives in these metrics.  Can we emit enough metrics to detect these two cases:

* A sign-in that didn't match `Account.history.seen`, but for which the sign-in confirmation was completed successfully.
* A sign-in that did match `Account.history.seen`, but for which the sign-in confirmation was flagged as "this is not me".

The geo-profiling feature doc has a lot of good work on clarifying the precise metrics here.